### PR TITLE
Added some padding to the left of the text - Inline Parameter Name Hints

### PR DESCRIPTION
**Pull request by akhera99**
Hello,

As per the request of the UX Board, I have added some small amount of padding between the TextBlock and the Border to make the text feel less cramped.

Before:
image

After:
image

### CODE REVIEW

**Comment by JoeRobich** [comment]
@akhera99 What is the goal of the padding? Just to bring it off the left side or is it to make the text appear centered?
image

**Comment by akhera99** [comment]
> @akhera99 What is the goal of the padding? Just to bring it off the left side or is it to make the text appear centered?
> image

The UX team said that, because the colon on the right leaves a lot of empty space, it makes it look more padded to the right. So I just added a little bit on the left to appear more balanced.

File: src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
                 FontSize = format.FontRenderingEmSize - (0.25 * format.FontRenderingEmSize),
                 FontStyle = FontStyles.Normal,
                 Foreground = format.ForegroundBrush,
-                Padding = new Thickness(0),
+                Padding = new Thickness(1, 0, 0, 0),
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs:42]: needs comment.

Comment (akhera99) [reply src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs]: Added

**Pull request merged**